### DISCUSSION TITLE
How to modify the value of UNIQUE KEY?

### DISCUSSION THREAD

**Discussion by Python-IBM** [apache/doris#25281] [A - General / Q&A]
Doris has generated a UNIQUE KEY, such as UNIQUE KEY (` id `, ` time `). How can I use SQL statements to modify the value of UNIQUE KEY, such as UNIQUE KEY (` time `, ` code `)?

**Comment by LemonLiTree** [comment]
create new table, insert into then replace

**Reply by Python-IBM** [reply]
There is data in the table, and I cannot create a new table. This will affect the data. Is there any other way?

**Reply by LemonLiTree** [reply]
The key column cannot be modified and the table can only be recreate.
